go/def.bzl

```diff
@@ -133,6 +133,14 @@ def go_environment_vars(ctx):
                                    {"GOOS": "linux",
                                     "GOARCH": "amd64"})
 
+def _tmp_cmd_file(cmds, ctx, fn):
```

Comment:
Can we call this function `emit_generate_params_action` or something?
I prefer the suffix `_action` to be consistent to other actions, and I prefer the name `params` to follow a convention in Bazel.